tests/transformers/test_transformers_model_export.py

```diff
@@ -1842,7 +1845,7 @@ def test_classifier_pipeline_pyfunc_predict(text_classification_pipeline):
     # validate that the pyfunc served model registers text_pair in the same manner as native
     for key in ["score", "label"]:
         for value in [0, 1]:
-            assert values_dict[key][value] == native_predict[value][key]
+            assert math.isclose(values_dict[key][value], native_predict[value][key], rel_tol=1e-7)
```

Comment:
this failure doesn't seem to repro locally for me. i think there might be some difference between how transformers serializes/loads the weights on linux. i'm trying to look into it but i'm not having much luck.

it's a little bit concerning that the prediction is not 100% the same, even if the difference is small, as i don't think there's supposed to be nondeterminism for this calculation. however, i'm not sure if it's something that we can fix from the mlflow side, or if it's a transformers thing.

i made this change to make the tests pass since the values are pretty close, but wanted to check if anyone thinks it's a serious problem.